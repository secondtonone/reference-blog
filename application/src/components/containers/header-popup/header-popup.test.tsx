/**
 * @jest-environment jsdom
 */
import renderer from 'react-test-renderer';
import HeaderPopup from '.';

jest.mock('~/components/containers/svg-icon', () => jest.fn().mockReturnValue('svg-icon'));

jest.mock('~hooks/use-with-popup', () => jest.fn(() => ({
  show: jest.fn(),
  hide: jest.fn(),
  isVisible: false,
  toggle: jest.fn()
})));

describe('Match snapshots HeaderPopup', () => {
  it('default', () => {
    const tree = renderer.create(
      <HeaderPopup name="content">Content</HeaderPopup>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
