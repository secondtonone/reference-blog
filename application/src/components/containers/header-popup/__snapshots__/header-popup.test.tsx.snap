// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshots HeaderPopup default 1`] = `null`;
